packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -177,6 +197,16 @@ const getOrphanPinpointApplications = async (account: AWSAccountInfo, region: st
   return apps;
 };
 
+/**
+ * Get all AppSync Apis in the account, and filter down to the ones we consider stale.
+ */
+const getOrphanAppSyncApis = async (account: AWSAccountInfo, region: string): Promise<AppSyncApiInfo[]> => {
+  const appSyncClient = new aws.AppSync(getAWSConfig(account, region));
+  const listApisResponse = await appSyncClient.listGraphqlApis({ maxResults: 1000 }).promise();
```

Comment:
1000 is a lot when querying by account + region; we could reduce to 50 and probably be OK - then we won't get throttled elsewhere

Reply:
adjusted.

Reply:
looks like 50 is still to big.
![image](https://user-images.githubusercontent.com/5849952/209738692-382c144c-dc84-4b92-9761-48ebb608b630.png)